platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/capabilities/CapabilitiesConflictResolutionIssuesIntegrationTest.groovy

```diff
@@ -856,6 +856,102 @@ class CapabilitiesConflictResolutionIssuesIntegrationTest extends AbstractIntegr
         }
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/35625")
+    def "can detect Slf4J logger implementation conflicts"() {
+        given:
+        buildFile << """
+            plugins {
+                id("org.gradlex.jvm-dependency-conflict-resolution").version("2.4")
```

Comment:
❓ Do we need all those tests with a dependency on a published plugin and real dependencies?
This might mean flakiness if the test runs and there are network issues.